### PR TITLE
change: skip p2/p3 tests in eu-central-1

### PR DESCRIPTION
**Pull request by jesterhazy**
*Description of changes:*

skip p2/p3 tests in eu-central-1 during continuous testing runs to avoid capacity issues

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#commit-message-guidlines)
- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=27cc93cc-a770-4719-8a8d-892c61bb8543%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=547c60a3-87a7-4273-940e-de9a06c950fa%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: tests/integ/test_tf_script_mode.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
+from __future__ import absolute_import
```
Comment (mvsusp) [inline tests/integ/test_tf_script_mode.py]: Duplicated

File: tests/integ/test_tf_script_mode.py [hunk 2]
```diff
@@ -22,9 +23,8 @@
 from sagemaker.tensorflow import TensorFlow
 from six.moves.urllib.parse import urlparse
 from sagemaker.utils import unique_name_from_base
-import tests.integ as integ
-from tests.integ import kms_utils
-import tests.integ.timeout as timeout
+
+import tests.integ
```
Comment (mvsusp) [inline tests/integ/test_tf_script_mode.py]: Nice. Another way I like to do as well is:
```python
from test import integ
```

File: tests/integ/test_tf_script_mode.py (hunk 1 above)
Comment (jesterhazy) [reply tests/integ/test_tf_script_mode.py]: thx will fix

File: tests/integ/test_tf_script_mode.py (hunk 2 above)
Comment (jesterhazy) [reply tests/integ/test_tf_script_mode.py]: that works too. i'm long winded so i generally prefer fully FQ names.

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=ddb2c03c-094e-4fe3-85fe-4350b86a0a5c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=9be3fdae-f59a-4c38-b84e-82c6dadcf730%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**